### PR TITLE
Update `atom.video.spec` to check consent is sent to YouTube

### PR DESCRIPTION
**Pull request by arelra**
## What does this change?

Updates `atom.video.spec` to check consent is sent to YouTube in both consented and reject-all flows.

## Why?

So we can catch inadvertent bugs that might cause consent not to be sent to YouTube.

### After

Consented:

Screenshot 2022-02-23 at 17 19 26

Reject All:

Screenshot 2022-02-23 at 13 55 06

### CODE REVIEW

File: dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js [hunk 1]
```diff
@@ -21,6 +22,71 @@ const interceptPlayEvent = (id) => {
 	);
 };
 
+const parseCustParams = (custParamsString) => {
+	const custParams = decodeURIComponent(custParamsString);
```
Comment (jamesgorrie) [inline dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js:26]: You can use [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) which has a nicer API (and we could remove this function).

**Review by oliverlloyd — APPROVED** [review]
Really great to get more coverage in this area 👍

**Review by jamesgorrie — APPROVED** [review]
Really nice to have certainty in this important area!

File: dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js (hunk 1 above)
Comment (arelra) [reply dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js]: Thanks for the suggestion! That works nicely for the `cust_params` field.

File: dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js [hunk 2]
```diff
@@ -21,6 +21,60 @@ const interceptPlayEvent = (id) => {
...
+				decodeURIComponent(adTagParameters.cust_params),
+			);
+			// check consent related properties
+			// cmpGdpr = consentState.tcfv2.gdprApplies
+			expect(adTagParameters.cmpGdpr, 'check GDPR applies').to.equal(1);
+			// cmpVcd = consentState.tcfv2.tcString
+			expect(adTagParameters.cmpVcd, 'check TCFV2 tcString').to.not.be
+				.undefined;
+			if (rejectAll) {
+				// user has NOT consented to all purposes
```
Comment (kenoir) [inline dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js:48]:
```suggestion
				// user has not given consent for any purpose
```

**Review by kenoir — APPROVED** [review]
Nice ✨

**Pull request merged**